06-building-a-program-start-to-finish/automating-tasks/Output/README.MD

```diff
@@ -0,0 +1,3 @@
+#### A screencast showing gulp watching and refreshing your browser.
+### Link to screencast
+[link to screen cast of gulp automation](http://youtu.be/DDMv22CdbmE?hd=1)
```

Comment:
I have confided in my colleagues and I have been able to record a screen cast showing gulp watching and refreshing my browser as expected.